documentation/docs/verifiable_credentials/create.mdx

```diff
@@ -12,15 +12,59 @@ keywords:
+
+### Proof Options {#proof-options}
+
+A digital signature on a verifiable credential both provides data integrity and proves the DID of the issuer.
+The proof section embedded in a credential may also include additional metadata.
+
+The following metadata properties can be configured by the framework and are optional and omitted by default:
+
+- **Created**: timestamp of when the credential was signed.
+- **Expires**: timestamp after which the credential is no longer considered valid.
+- **Challenge**: arbitrary random string. Sent by the issuer and mitigates replay attacks; should be sufficiently random and uniquely generated per credential request.
+- **Domain**: arbitrary string. Sent by the issuer and can help mitigate replay attacks when used with a challenge.
+- **Proof Purpose**: indicates the purpose of the signature.
+  - **AssertionMethod**: to assert a claim. The signing verification method must have an [`assertionMethod`](https://www.w3.org/TR/did-core/#assertion) relationship to be valid.
+  - **Authentication**: to authenticate the signer. The signing verification method must have an [`authentication`](https://www.w3.org/TR/did-core/#authentication) relationship to be valid.
+
+### Credential Validation {#credential-validation}
 
-This example shows how you can create a Verifiable Credential and validate it. In this example, Alice takes the role of the subject, while we also have an issuer.
-The issuer signs a UniversityDegreeCredential type verifiable credential with Alice's name and DID.
-This Verifiable Credential can be verified by anyone, allowing Alice to take control of it and share it with anyone.
+After creating a verifiable credential, and before sending the VC to the subject, an issuer may want to make sure certain properties of the VC are valid. 
+The Iota Framework allows issuers to perform some validations using the CredentialValidator.
+
+The following checks can be verified by the validator:
+
+- **Credential proofs**: verifies the credential signatures against the DID Documents of the issuer.
+- **Semantic Validation**: ensures the credential adhere to the specification.
+- **Subject Holder Relationship**: ensures that the credential relationship between the holder and the subject match.
+- **Optional validations**: additional checks on signatures and credential fields can be configured.
+- **Latest Issuance date**: check that the [`issuanceDate`](https://www.w3.org/TR/vc-data-model/#issuance-date) property, if present, is not after a specific datetime. Defaults to the current datetime if unset.
+- **Earliest Expiry date**: check that the [`expirationDate`](https://www.w3.org/TR/vc-data-model/#expiration) property, if present, is not before a specific datetime. Defaults to the current datetime if unset.
+
+The last two checks listed above are specified using CredentialValidationOptions.
+
```

Comment:
I think we need to start linking articles a bit stronger together. Lets add a small section / paragraph / Note here on "sharing Verifiable Credentials" and mention that it should be done in the format of a VP and link to the article.